### DISCUSSION TITLE
Issue with the Industry sessions and self-learning

### DISCUSSION THREAD

**Discussion by AmriteshATiwari** [github/india#543] [Students]
I have completed the industry sessions but the completion status has not been updated and each video loops when it gets over resulting in the non completion. I have faced this issue for about 3 weeks now and I am now unable to access the project section.
Please tell me what can be done about this.

**Comment by aparidhi9** [comment]
Yes, Same problem arise with me and I am also not able to access with self learning courses.
Please guide.
